packages/react-ui-core/src/Ratings/SampleSvgs/Circle.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { randomId } from '@rentpath/react-ui-utils'
+import LinearGradient from '../LinearGradient'
+
+export default class Circle extends Component {
+  static propTypes = {
+    id: PropTypes.string,
+    theme: PropTypes.object,
+    backgroundColor: PropTypes.string,
+    stroke: PropTypes.string,
+    strokeWidth: PropTypes.number,
+    width: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+    width: '0%',
+  }
+
+  render() {
+    const {
+      id,
+      backgroundColor,
+      width,
+    } = this.props
+
+    return (
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="50"
+          height="50"
+          viewBox="0 0 600 300"
+        >
+          <LinearGradient
+            id={id}
+            width={width}
+            backgroundColor={backgroundColor}
+          />
+          <circle cx="250" cy="150" r="145" fill="url(#lg)"
```

Comment:
Please fix formatting for this circle;  indentation needs adjusted and  props should be on their own lines.